src/PlatformSwitch/PlatformSwitch.tsx

```diff
@@ -157,6 +158,57 @@ export const PlatformSwitch: FC<PlatformSwitchProps> = ({
     />
   ));
 
+  if (isMobile) {
+    const SelectedIcon = PlatformIcons[selected || 'Builder'];
+    return (
+      <Box sx={{ position: 'relative' }}>
+        <SpeedDial
+          ariaLabel='SpeedDial basic example'
+          sx={{
+            position: 'absolute',
+            '& button': {
+              width: '40px',
+              height: '40px',
+            },
+          }}
+          icon={selected && <SelectedIcon />}
```

Comment:
In order to make the speed dial more visible on the blue banner you can add a white border to the speed dial button. This has the added advantage of matching the current platform design which has a border.
```suggestion
          }}
          FabProps={{ size: 'small', sx: { border: '2px solid white' } }}
          icon={selected && <SelectedIcon />}
```